src/cli/config.rs

```diff
@@ -143,7 +146,12 @@ pub fn load_config(opt: &Opt) -> Result<Config> {
 
                     // Fallback to a default configuration
                     debug!("config: falling back to default config");
-                    Ok(Config::default())
+                    let config = Config::default();
+
+                    #[cfg(feature = "editorconfig")]
+                    load_editorconfig(config, &current_dir);
```

Comment:
Is the output of this actually being used, since `config` isn't mutable?

Reply:
It was used by `use_fallbacks`, which is now removed.